Cleanup get-workflow-job-id action

malfet opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #92193

To be landed few days later then rest of the changes

As workflow can never fail now, no need to retry it

malfet committed:
Cleanup get-workflow-job-id action

To be landed few days later then rest of the changes

As workflow can never fail now, no need to retry it

[ghstack-poisoned]

huydhn reviewed (approved):
LGTM! FYI I have https://github.com/pytorch/pytorch/pull/92183 coming up next to use conda python instead of system python so that all dependencies are already there.  I'll rebase mine on top of this change once this is merged

malfet committed:
Update on "Cleanup get-workflow-job-id action"


To be landed few days later then rest of the changes

As workflow can never fail now, no need to retry it

[ghstack-poisoned]

malfet committed:
Update on "Cleanup get-workflow-job-id action"


To be landed few days later then rest of the changes

As workflow can never fail now, no need to retry it

[ghstack-poisoned]

malfet committed:
Update on "Cleanup get-workflow-job-id action"


To be landed few days later then rest of the changes

As workflow can never fail now, no need to retry it

[ghstack-poisoned]

malfet committed:
Update on "Cleanup get-workflow-job-id action"


To be landed few days later then rest of the changes

As workflow can never fail now, no need to retry it

[ghstack-poisoned]

malfet committed:
Update on "Cleanup get-workflow-job-id action"


To be landed few days later then rest of the changes

As workflow can never fail now, no need to retry it

[ghstack-poisoned]

malfet committed:
Update on "Cleanup get-workflow-job-id action"


To be landed few days later then rest of the changes

As workflow can never fail now, no need to retry it

[ghstack-poisoned]

malfet committed:
Update on "Cleanup get-workflow-job-id action"


To be landed few days later then rest of the changes

As workflow can never fail now, no need to retry it

[ghstack-poisoned]

malfet committed:
Update on "Cleanup get-workflow-job-id action"


To be landed few days later then rest of the changes

As workflow can never fail now, no need to retry it

[ghstack-poisoned]

malfet commented:
@pytorchbot merge
